config/samples/flows_v1alpha1_flowcollector.yaml

```diff
@@ -71,6 +77,13 @@ spec:
         certFile: user.crt
         certKey: user.key
   loki:
+    instanceSpec:
+      enable: false
+      secretName: loki-secret
+      objectStorageType: s3
+      size: 1x.extra-small
+      storageClassName: gp2
+      retentionDays: 1
```

Comment:
Must it be in days? maybe if the storage is too high, some users might want to set the retention to hours.

Reply:
Yes it is in days on loki-operator side. We can sync with logging team if we really need less